ambry-replication/src/main/java/com/github/ambry/replication/BackupCheckerThread.java

```diff
@@ -168,23 +185,23 @@ protected void fixMissingStoreKeys(
    * @param acceptableStoreErrorCodes Acceptable error codes when retrieving the blob from local-store
```

Comment:
nit: you need to edit javadocs.

Reply:
Why do you need `MessageInfoType` when `MessageInfo` is already passed as argument.

Reply:
1/ what shd i add to javadocs ?
2/ yeah, removed.